articles/aks/open-service-mesh-troubleshoot.md

```diff
@@ -185,59 +185,6 @@ A well configured Mutating Webhook Configuration would look exactly like this:
 }
 ```
 
-### Check whether OSM Controller has given the Validating (or Mutating) Webhook a CA Bundle
```

Comment:
In the above section we will need to add a check for `osm-bootstrap` just like osm-controller and osm-injector. The number of injector and controller pods will be 2, so the above sections need to be update accordingly